packages/repository-json-schema/src/build-schema.ts

```diff
@@ -15,6 +15,7 @@ import {
 import * as debugFactory from 'debug';
 import {JSONSchema6 as JSONSchema} from 'json-schema';
 import {JSON_SCHEMA_KEY, MODEL_TYPE_KEYS} from './keys';
+const utils = require('../../cli/lib/utils');
```

Comment:
We cannot use `utils` from `@loopback/cli` as it's not a dependency of this module.